server/models/Appointment.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const appointmentSchema = new mongoose.Schema({
+  meeting_id: {
+    type: String,
```

Comment:
type should be of _mongoose.Types.ObjectId_ , which allows you to populate meeting details easily